storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManager.java

```diff
@@ -378,7 +377,7 @@ private void initializeResources() {
         long startTimeMs = time.milliseconds();
         Admin adminClient = null;
         try {
-            adminClient = AdminClient.create(rlmmConfig.commonProperties());
+            adminClient = Admin.create(rlmmConfig.commonProperties());
```

Comment:
Thank you for this, can you also change the use of `AdminClient.create(Properties props)` .

Reply:
Thanks for the heads up, I've made changes to all the code that uses `AdminClient.create(Properties props) `